lib/themes/dosomething/paraneue_dosomething/js/utilities/Globe.js

```diff
@@ -5,7 +5,14 @@ function init() {
   const globeUpdateUrl = setting('dosomethingSetting.globeUrl');
   if (globeUpdateUrl !== undefined) {
     const countryCode = setting('dosomethingSetting.countryCode');
+
+    // Prevents sensitive information from leaking to 3rd party in the referer.
+    let $meta = $('<meta name="referrer" content="no-referrer" />');
```

Comment:
I figured we were ok having a referrer most of the time. It's just whenever we reach out to 3rd party services it could be a problem.